src/client/styles/scss/_page-history.scss

```diff
@@ -0,0 +1,81 @@
+.revision-history {
```

Comment:
The "revision-history" class did not exist.
It seems to be a style that was used in the past(https://github.com/weseek/growi/commit/bcc5733934ad9afd95f106d55c4f3c098d0a3c06), but I used it this PR because it was available.